protos/dcs/common/v0/common.proto

```diff
@@ -201,22 +201,17 @@ message Unit {
   Coalition coalition = 4;
   // The DCS type-name of the unit. e.g "MiG-29A", "ZSU_57_2" or "Hawk ln"
   string type = 5;
-  // The position of the unit
-  Position position = 6;
+  // Information about the unit in 3D space.
+  Transform transform = 6;
   // The name of the player if one is in control of the unit
   optional string player_name = 7;
   // The name of the group that the unit belongs to
   string group_name = 8;
   // The number of this unit in the group. Does not change as units are
   // destroyed
   uint32 number_in_group = 9;
-  // The horizontal speed of the unit. If it is doing mach one straight up then
-  // the speed will be 0
-  double speed = 10;
-  // The heading of the unit
-  double heading = 11;
   // The group category.
-  GroupCategory category = 12;
+  GroupCategory category = 10;
```

Comment:
This is a breaking change that might effect some users of DCS-gRPC.

Reply:
Luckily we are in the 0.x.x 😁